lib/types/types.ts

```diff
@@ -0,0 +1,74 @@
+export interface UserData {
```

Comment:
You can use @internxt/sdk type to get this and you don't need to maintain this https://github.com/internxt/sdk/blob/master/src/shared/types/userSettings.ts#L5